xml/System.Windows.Media.Effects/BitmapEffect.xml

```diff
@@ -163,7 +163,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Creates a handle to an [IMILBitmapEffect](http://msdn.microsoft.com/library/4bd5697d-bdde-4485-a05b-b3fea1403e16) object that is used to initialize a custom effect.</summary>
+        <summary>Creates a handle to an <see href="">IMILBitmapEffect</see> object that is used to initialize a custom effect.</summary>
```

Comment:
The URL is missing here; it's <https://docs.microsoft.com/en-us/windows/desktop/api/mileffects/nn-mileffects-imilbitmapeffect>.